packages/util/src/string.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { indent, trim, encodeBase64 } from './string';
+
+describe('encodeBase64', () => {
+  it('encodes a string', () => {
+    expect(encodeBase64('test')).toMatchInlineSnapshot('"dGVzdA=="');
```

Comment:
use this method in the `basicHeader` method.

Reply:
Will do, just fyi, this was here before, but not tested.

Reply:
Done.